pkg/proxy/service.go

```diff
@@ -182,14 +182,14 @@ func (sct *ServiceChangeTracker) newBaseServiceInfo(port *v1.ServicePort, servic
 
 	// Log the IPs not matching the ipFamily
 	if ips, ok := ipFamilyMap[utilproxy.OtherIPFamily(sct.ipFamily)]; ok && len(ips) > 0 {
-		klog.V(4).Infof("service change tracker(%v) ignored the following external IPs(%s) for service %v/%v as they don't match IPFamily", sct.ipFamily, strings.Join(ips, ","), service.Namespace, service.Name)
+		klog.V(4).InfoS("Service change tracker ignored the following external IPs for given service as they don't match IPFamily", "serviceChangeTracker", sct.ipFamily, "externalIPs", strings.Join(ips, ","), "service", klog.KObj(service))
```

Comment:
the  key for  ipFamily can be "ipFamily"

Reply:
Okay sure, fixing it